br/pkg/lightning/common/util.go

```diff
@@ -616,6 +616,15 @@ func GetBackoffWeightFromDB(ctx context.Context, db *sql.DB) (int, error) {
 	return strconv.Atoi(val)
 }
 
+// GetExplicitRequestSourceTypeFromDB gets the explicit request source type from database.
+func GetExplicitRequestSourceTypeFromDB(ctx context.Context, db *sql.DB) (string, error) {
+	val, err := getSessionVariable(ctx, db, variable.TiDBExplicitRequestSourceType)
+	if err != nil {
+		return "", err
+	}
+	return val, nil
```

Comment:
```suggestion
	return getSessionVariable(ctx, db, variable.TiDBExplicitRequestSourceType)
```
BTW, please ensure the extra logic still compatible with old tidb that doesn't have this variable.

Reply:
I think if old tidb doesn't have this variable, there just returns"". And will pass "" to context which will be replaced `default` in client-go. Maybe not have a compatible problem?